Etabs_Adapter/_Engine/Convert/ToCSI.cs

```diff
@@ -102,18 +102,22 @@ public static eMatType ToCSI(this MaterialType materialType)
                     return eMatType.Steel;
                 case MaterialType.Concrete:
                     return eMatType.Concrete;
-                case MaterialType.Timber://no material of this type in ETABS !!! 
-                    return eMatType.Steel;
+                case MaterialType.Timber:
+                    Engine.Reflection.Compute.RecordWarning("ETABS does not contain a definition for Timber materials, the material has been set to type 'Other' with 'Orthotropic' directional symmetry");
+                    return eMatType.NoDesign;
                 case MaterialType.Rebar:
                     return eMatType.Rebar;
                 case MaterialType.Tendon:
                     return eMatType.Tendon;
-                case MaterialType.Glass://no material of this type in ETABS !!!
-                    return eMatType.Steel;
-                case MaterialType.Cable://no material of this type in ETABS !!!
+                case MaterialType.Glass:
+                    Engine.Reflection.Compute.RecordWarning("ETABS does not contain a definition for Glass materials, it has been set as an equivilant orthotropic material instead");
+                    return eMatType.NoDesign;
+                case MaterialType.Cable:
+                    Engine.Reflection.Compute.RecordWarning("ETABS does not contain a definition for Cable materials, it has been set as an steel material instead");
                     return eMatType.Steel;
                 default:
-                    return eMatType.Steel;
+                    Engine.Reflection.Compute.RecordWarning("BHoM material type not found, it has been set as an equivilant orthotropic material instead");
```

Comment:
Same here. Also, would not say anything about orthotropic for this case, as at this point we will not be sure that that will happen.